pkg/cmd/cmd.go

```diff
@@ -49,7 +50,7 @@ func setupLogger(*cobra.Command, []string) {
 	}
 }
 
-func NewObsctlCmd(ctx context.Context) *cobra.Command {
+func NewObsctlCmd(ctx context.Context, path ...string) *cobra.Command {
```

Comment:
Optional suggestion: I'm wondering if instead of adding new param everywhere we could use env var to optionally override the path.

Reply:
We can! I was keeping it as a variable as it's only ever used for tests and doesn't have much utility for users as it isn't exposed. Do you think this should be a user-facing option? 🙂

Reply:
Shouldn't uses eventually have the option to override it as well? But I'm not sure honestly, I just didn't like adding extra parameter (= extra noise) everywhere just to satisfy testing.

Reply:
Let me change it to an env variable then!

Reply:
Done in https://github.com/observatorium/obsctl/pull/12/commits/fa96095811d3ffdba2ee480851d876c913effa97